Spotify Bluetooth Headset Listener.xcodeproj/project.pbxproj

```diff
@@ -398,7 +398,7 @@
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = "Spotify Bluetooth Headset Listener/Spotify Bluetooth Headset Listener-Prefix.pch";
 				INFOPLIST_FILE = "Spotify Bluetooth Headset Listener/Spotify Bluetooth Headset Listener-Info.plist";
-				MACOSX_DEPLOYMENT_TARGET = 10.6;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Your call, but I had to do this to get it to work.

Perhaps a second branch and two builds? One for Yosemite and one for
before.
On 16 Mar 2015 00:29, "Josh Guice" <notifications@github.com> wrote:

> In Spotify Bluetooth Headset Listener.xcodeproj/project.pbxproj
> <https://github.com/jguice/mac-bt-headset-fix/pull/21#discussion_r26454689>
> :
>
> > @@ -398,7 +398,7 @@
> >  				GCC_PRECOMPILE_PREFIX_HEADER = YES;
> >  				GCC_PREFIX_HEADER = "Spotify Bluetooth Headset Listener/Spotify Bluetooth Headset Listener-Prefix.pch";
> >  				INFOPLIST_FILE = "Spotify Bluetooth Headset Listener/Spotify Bluetooth Headset Listener-Info.plist";
> > -				MACOSX_DEPLOYMENT_TARGET = 10.6;
> > +				MACOSX_DEPLOYMENT_TARGET = 10.10;
>
> I don't think we want to change this. It is the *minimum* version of Mac
> OS X required to run the app ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jguice/mac-bt-headset-fix/pull/21/files#r26454689>.
>

-- 
e: *al@perchten.co.uk <al@perchten.co.uk>*
t:  *07779266625*
w: *http://perchten.co.uk <http://perchten.co.uk/>*